src/BotwinExtensions.cs

```diff
@@ -148,9 +147,11 @@ private static void ApplyGlobalBeforeHook(IApplicationBuilder builder, BotwinOpt
         /// </summary>
         /// <param name="services">The <see cref="IServiceCollection"/> to add Botwin to.</param>
         /// <param name="assemblies">Optional array of <see cref="Assembly"/> to add to the services collection. If assemblies are not provided, Assembly.GetEntryAssembly is called.</param>
-        public static void AddBotwin(this IServiceCollection services, params Assembly[] assemblies)
+        public static void AddBotwin(this IServiceCollection services)
```

Comment:
We should probably provide an overload so you can specify the assembly your modules belong in explicitly.

Reply:
Actually, I don't think you do need this